src/pages/ws/ui/variant-drawer/variant-drawer.store.ts

```diff
@@ -89,6 +81,9 @@ class VariantDrawerStore {
           case VariantDrawerPredefinedPresets.List:
             this.currentGridLayout = []
             break
+          case VariantDrawerPredefinedPresets.Preset1:
+            this.currentGridLayout = preset.layout
+            break
         }
       } else if (preset.layout) {
         this.currentGridLayout = cloneDeep(preset.layout)
```

Comment:
maybe just `this.currentGridLayout = preset.layout ? cloneDeep(preset.layout) : []` without ifs and the switch case block?